Cargo.toml

```diff
@@ -17,6 +22,11 @@ serde = "1.0.203"
 thiserror = "1.0.61"
 tracing = "0.1.40"
 url = "2.5.2"
+# zkSync types
+zksync_types = { git = "https://github.com/matter-labs/zksync-era.git", rev = "1bfff0e007e2fb5a4b4b885cf5c69a5cd290888b" }
+zksync_web3_decl = { git = "https://github.com/matter-labs/zksync-era.git", rev = "1bfff0e007e2fb5a4b4b885cf5c69a5cd290888b", features = [
```

Comment:
No, we don't want to have these dependencies in the `alloy-zksync` -- the implementation should be free of any of "main" ZKsync code.

These deps are _very_ heavyweight, and not optimized to be used as a part of the library. In fact, likely the transition will be opposite: the "main" codebase will eventually depend on this crate (as well as on `alloy`).